digicampipe/image/sky_image.py

```diff
@@ -0,0 +1,543 @@
+            rect = Rectangle(crop_origin, width=rect.width(), height=rect.height(), fill=False, color='y', linestyle='dashdot')
+            ax.add_artist(rect)
+        # plt.imshow(image_treated, cmap='gray')
+        plt.plot(0, 0, 'w+')
+        plt.grid(None)
+        plt.axis('off')
+        ax.get_xaxis().set_visible(False)
+        ax.get_yaxis().set_visible(False)
+        output_filename = self.filename.replace('.fits', '-treated.png')
+        plt.savefig(output_filename, bbox_inches='tight', pad_inches=0)
+        plt.close(fig)
+        print(output_filename, 'saved.')
+
+
+class LidCCDObservation:
+    """
+    LidCCDObservation regroups several LidCCDImages. Useful to find moving stars.
+    """
+    def __init__(self, filenames, rectangles, threshold=None,
+                 scale_low_images_deg=None, scale_high_images_deg=None,
+                 guess_ra_dec=None, guess_radius=None, method='local'):
+        """
+        Create a lid ccd observation from several pictures of the lid CCD camera.
+        Pointing is determined for each of the regions defined.
+        :param filenames: list of strings containing the path to lid CCD images (fits files).
+        :param rectangles: list of rectangles defining the regions of interest in pixels
+        :param threshold: threshold: pixels with a smaller value than threshold after subtraction are set to 0. Default at 100x the
+            average pixel value after subtraction.
+        :param scale_low_images_deg: minimum field of view (in degrees) considered during fitting.
+        :param scale_high_images_deg: maximum field of view (in degrees) considered during fitting.
+        :param guess_ra_dec: aproximate pointing coordinates
+        :param guess_radius: radius of pointing  coordinates around guess_ra_dec tried during determination
+        :param method: if method is "local", we look for local installation of astrometry.net, otherwise we use web service.
+        """
+        self.lidccd_images = []
+        image_shape = None
+        if len(filenames) == 0:
+            raise AttributeError('filenames must be a non-empty list')
+        for filename in filenames:
+            lidccd_image = LidCCDImage(filename, rectangles=rectangles,
+                                       scale_low_images_deg=scale_low_images_deg,
+                                       scale_high_images_deg=scale_high_images_deg,
+                                       guess_ra_dec = guess_ra_dec,
+                                       guess_radius = guess_radius)
+            if image_shape is None:
+                image_shape = lidccd_image.image_shape
+            elif image_shape != lidccd_image.image_shape:
+                raise AttributeError('all images must have the same size')
+            self.lidccd_images.append(lidccd_image)
+        static_images = []
+        for crop_area_index, crop_area_shape in enumerate(lidccd_image.sky_images_shape):
```

Comment:
Here `liccd_image` is just the last image in the list of images, is this intended? It looks like a bug to me